src/simulation/LoadGenerator.cpp

```diff
@@ -415,7 +415,7 @@ LoadGenerator::start(GeneratedLoadConfig& cfg)
     mStartTime =
         std::make_unique<VirtualClock::time_point>(mApp.getClock().now());
 
-    releaseAssert(!mPreLoadgenApplySorobanSuccess);
+    releaseAssert(mPreLoadgenApplySorobanSuccess == 0);
     releaseAssert(!mPreLoadgenApplySorobanFailure);
```

Comment:
Nit: integer compare instead of implicit bool cast here too.

Reply:
Fixed in 98d1ca84b359685dbb164bacaa4c63e628c92df6